src/themes/default/components/core/blocks/Auth/ForgotPass.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div>
+    <div class="py35 px55 bg-lightgray">
+      <h1 class="my0">Forgot your password?</h1>
+    </div>
+    <div class="py35 px55 bg-white c-gray">
+      <form>
+        <div class="mb35">
+          <p>Enter your email adress below and we'll send you password reset instructions.</p>
+          <input type="email" name="email" v-model="email" placeholder="E-mail address *">
+          <span class="validation-error" v-if="!$v.email.required">Field is required.</span>
+          <span class="validation-error" v-if="!$v.email.email">Please provide valid e-mail address.</span>
+        </div>
+        <div class="mb35">
+          <button-full class="btn-full p0" text="Reset password" @click.native="sendEmail"></button-full>
+        </div>
+      </form>
+    </div>
+  </div>
+</template>
+
+<script>
+import { coreComponent } from 'lib/themes'
+import EventBus from 'src/event-bus/event-bus'
+
+import ButtonFull from 'theme/components/theme/ButtonFull.vue'
+import { required, email } from 'vuelidate/lib/validators'
+
+export default {
+  data () {
+    return {
+      email: ''
+    }
+  },
+  validations: {
+    email: {
+      required,
+      email
+    }
+  },
+  methods: {
+    sendEmail () {
+      // todo: send email with reset password instructions
+
+      if (this.$v.$invalid) {
+        EventBus.$emit('notification', {
+          type: 'error',
+          message: 'Please fix the validation errors',
+          action1: { label: 'OK', action: 'close' }
+        })
+        return
+      }
+
+      EventBus.$emit('notification', {
+        type: 'success',
+        message: `An email has been sent to '${this.email}' with further informations on how to reset your password.`,
+        action1: { label: 'OK', action: 'close' }
+      })
+      this.$store.commit('ui/setSignUp', false)
+    }
+  },
+  mixins: [coreComponent('core/blocks/Auth/ForgotPass')],
+  components: {
+    ButtonFull
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+  @import '../../../../css/text.scss';
```

Comment:
same as in https://github.com/DivanteLtd/vue-storefront/pull/365

Reply:
Done